admin/coupon_admin.php

```diff
@@ -1331,11 +1336,14 @@ function check_form(form_name) {
 
     $cc_split = new splitPageResults($_GET['page'], $maxDisplaySearchResults, $cc_query_raw, $cc_query_numrows);
     $cc_list = $db->Execute($cc_query_raw);
+    if ($cc_list->EOF && (empty($_GET['cid']) || ($_GET['cid'] == $cc_list->fields['coupon_id'])) && empty($cInfo)) {
+      $cInfo = new objectInfo($cc_list->fields);
+    }
     while (!$cc_list->EOF) {
-      if (((!$_GET['cid']) || (@$_GET['cid'] == $cc_list->fields['coupon_id'])) && (!$cInfo)) {
+      if ((empty($_GET['cid']) || ($_GET['cid'] == $cc_list->fields['coupon_id'])) && empty($cInfo)) {
         $cInfo = new objectInfo($cc_list->fields);
       }
-      if ( (is_object($cInfo)) && ($cc_list->fields['coupon_id'] == $cInfo->coupon_id) ) {
+      if ( isset($cInfo) && (is_object($cInfo)) && ($cc_list->fields['coupon_id'] == $cInfo->coupon_id) ) {

```

Comment:
Can the `is_object($cInfo)` be replaced by `isset($cInfo)`?
(I recall when we added the `is_object()` check that it was for basically the same goal, just didn't use isset at the time.)